journal/modernized/18-chapter-17.adoc

```diff
@@ -248,8 +251,9 @@ and the next day to Moneygoff about forty miles; and so to Port Patrick,
 where I sent back my horse with John Rutledge and Henry Iveson,
 two Friends who came from Cumberland to accompany us to this place.
 
-Next day we took boat, and on the day following landed at Dunnaghadee in Ireland,
-being the 3d day of the seventh month, 1697.
+Next day we took a boat, and on the day following landed at Dunnaghadee in Ireland,
+being the 3rd day of the seventh month, 1697.
+//Do you want ordinal numbers to remain as is? I saw they are spelled out in the PDF.
```

Comment:
Again, this is also only spelled out in one of the google PDF versions, but not in the other. I don't think I have a preference here... although I think _most_ of the journals don't spell out words like this, and simply put 2nd or 3rd.